src/static-landing/REAMDME.md

```diff
@@ -0,0 +1,9 @@
+# Static Landing Page
```

Comment:
Typo in filename

Reply:
Fixed in https://github.com/bcgov/bcparks.ca/pull/235/commits/015285580ffa1c61e340a9ad6bf9cded18119500